vue-frontend/src/pages/contributions.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <div>
+    <metainfo>
+      <template v-slot:title="{ content }">
+        {{ content }}
+      </template>
+    </metainfo>
+    <Header />
+    <div class="tw-overflow-hidden">
+      <MobileLandingSection class="tw-block md:tw-hidden" />
+      <DesktopLandingSection class="tw-hidden md:tw-block" />
+    </div>
+    <NewFooter />
+  </div>
+</template>
+
+<script>
+import Header from "@/components/partials/NewHeader.vue";
+import MobileLandingSection from "@/components/contribution/MobileLanding.vue";
+import DesktopLandingSection from "@/components/contribution/DesktopLanding.vue";
+import config from "@/config.js";
+
+import { useMeta } from "vue-meta";
+import { defineAsyncComponent } from "vue";
+const NewFooter = defineAsyncComponent(() =>
+  import("@/components/partials/NewFooter.vue")
+);
+export default {
+  setup() {
+    var seoData = config.CONTRIBUTION_SEO_META_DATA;
+    useMeta({
+      meta: [
+        {
+          name: "robots",
+          content: "noindex, nofollow",
+          vmid: "robots",
+        },
+      ],
+      title: seoData.title,
+      description: seoData.description,
+      og: {
+        type: seoData.type,
+        title: seoData.title,
+        url: seoData.url,
+        image: seoData.image,
+      },
+    });
+  },
+  components: {
+    Header,
+    MobileLandingSection,
+    DesktopLandingSection,
+    NewFooter,
+  },
+  beforeRouteEnter(to, from, next) {
+    if (config.SHOW_CONTRIBUTION_PAGE === false) {
```

Comment:
I think the boolean variable doesn't need to match values using `===`, it can just be `!config.SHOW_CONTRIBUTION_PAGE`.